examples/07_advanced/plot_localizer_mass_univariate_methods.py

```diff
@@ -71,56 +71,107 @@
 
 ##############################################################################
 # Perform massively univariate analysis with permuted OLS
-neg_log_pvals_permuted_ols, _, _ = permuted_ols(
-    tested_var, fmri_masked,
+#
+# This method will produce both voxel-level FWE-corrected -log10 p-values and
+# :term:`TFCE`-based FWE-corrected -log10 p-values.
+neg_log_pvals_permuted_ols, _, _, neg_log_pvals_tfce = permuted_ols(
+    tested_var,
+    fmri_masked,
     model_intercept=True,
-    n_perm=5000,  # 5,000 for the sake of time. Idealy, this should be 10,000
-    verbose=1, # display progress bar
-    n_jobs=1)  # can be changed to use more CPUs
+    masker=nifti_masker,
+    tfce=True,
+    n_perm=1000,  # 1000 for the sake of time. Ideally, this should be 10000.
```

Comment:
The example is taking 20min to run on my box. This can be reduced by playing with n_jobs, but this is too much. I'm afraid we should further reduce the number of permutations to 400 or even 200 to make it more reasonable.

Reply:
Can do!